keyring.php

```diff
@@ -356,26 +356,26 @@ static function unpack_state_parameters( $state ) {
 	}
 
 	/**
-	 * Get an sha256 hash of a seialized array of parameters
+	 * Get an sha256 hash of a JSON encoded array of parameters
 	 *
-	 * @param string $serialized_parameters A serialized string of a parameter array.
+	 * @param string $encoded_parameters A JSON encoded string of a parameter array.
 	 * @return string An sha256 hash
 	 */
-	static function get_parameter_hash( $serialized_parameters ) {
-		return hash_hmac( 'sha256', serialize( $serialized_parameters ), NONCE_KEY );
+	static function get_parameter_hash( $encoded_parameters ) {
+		return hash_hmac( 'sha256', json_encode( $encoded_parameters ), NONCE_KEY );
```

Comment:
I don't think this extra `json_encode()` is necessary, since the parameters are already being passed as a JSON-encoded string.

Reply:
True, tidied this up.